calico_node/filesystem/sbin/start_runit

```diff
@@ -10,7 +10,7 @@ then
     exit $retval
 fi
 
-# Source any additional environment that was added by the startup script
-. ./startup.env
+# Source the nodename set by the startup script. 
```

Comment:
Not: you're not really "sourcing" it (which means executing it as a shell script in the current shell context) any more.